keywords.txt

```diff
@@ -103,6 +103,17 @@ MY_NODE_LOCK_COUNTER_MAX	LITERAL1
 MY_HW_HAS_GETENTROPY	LITERAL1
 hwGetentropy	KEYWORD2
 
+# TinyGSM
+MY_GATEWAY_TINYGSM  LITERAL1
```

Comment:
The Arduino IDE won't tolerate any whitespace except a single TAB between the keyword and LITERAL1. This code has spaces.

Reply:
This is very odd as I’m specifically using tab. Might be PHPStorm not playing nice. I’ll look into it.